site/docs/features/management.md

```diff
@@ -5,34 +5,107 @@ service can be scheduled and Nessie reports the outcome of each scheduled operat
+| GcOptions.getBloomFilterCapacity | 10000000 | Size of bloom filter for identification of referenced values |
+| GcOptions.getMaxAgeMicros | 7 days | age at which a commit starts to expire |
+| GcOptions.getTimeSlopMicros | 1 day | minimum age a values can be before it will be considered expired |
+| GcActionsConfig.getDynamoRegion | provider default | AWS Region of the Nessie DynamoDB |
+| GcActionsConfig.getDynamoEndpoint | provider default | Custom AWS endpoint of the Nessie DynamoDB |
+| GcActionsConfig.getStoreType | DYNAMO | only backend which supports GC |
+
+Running the action can be done simply by:
+```java
+    GcActions actions = new GcActions.Builder(spark)
+                                     .setActionsConfig(actionsConfig)
+                                     .setGcConfig(gcOptions)
+                                     .setTable(TABLE_IDENTIFIER).build(); // (1)
+    Dataset<Row> unreferencedAssets = actions.identifyUnreferencedAssets(); // (2)
+    actions.updateUnreferencedAssetTable(unreferencedAssets); // (3)
+```
+The first step above builds the action with known configs. Step 2 generates a DataFrame of unreferenced assets and
+Step 3 writes it as an iceberg table.
+
+### Delete Unreferenced Assets
+
+The destructive garbage collection step is also a Spark job and takes as input the table that has been built above. This job
+is modelled as an Iceberg Action and has a similar API to the other Iceberg Actions. In the future it will be registered with
+Iceberg's Action APIs and callable via Iceberg's custom [SQL statements](http://iceberg.apache.org/spark-procedures/).
+
+This Iceberg Action looks at the generated table from the 'Identify' step and counts the number of times a distinct asset has been
+seen. Effectively it performs a group-by and count on this table. If the count of an asset is over a specified threshold **AND** it
+was seen in the last run of the 'Identify' stage it is collectable. This asset is then deleted permanently. A report table of
+deleted object is returned to the user and either the records are removed from the 'identify' table or the whole table is purged.
+
+#### Configuration and running
+
+The relevant configuration items are:
+| parameter | default value | description |
+|---|---|---|
+| seenCount | 10 | How many times an asset has been seen as unreferenced in order to be considered for deletion |
+| deleteOnPurge | true | Delete records from the underlying iceberg table of unreferenced assets |
+| dropGcTable | true | Drop the underlying iceberg table or attempt to clean only the missing rows |
+| table | `null` | The iceberg `Table` which stores the list of unreferenced assets |
+
+Running the action can be done simply by:
+
+```java
+    Table table = catalog.loadTable(TABLE_IDENTIFIER);
+    new GcTableCleanAction(table, spark).dropGcTable(true).deleteCountThreshold(2).deleteOnPurge(true).execute();
```

Comment:
Does `execute()` return some report or summary?

Reply:
yes, added that in. The standard for Iceberg actions is to just return a few summary values, otherwise I would return the full list of deleted files.